apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -11,7 +11,7 @@ import {
   CommandLineStringParameter,
```

Comment:
> This is useful code for working with a Rush repo, so instead of having API users reinvent it, it's useful to just provide it. For example, I'm working on a project to filter integration tests that are run by a service external to the build by the set of projects that are changed in a PR. The code that `rush change` uses to determine which projects where changed is ideal for this.

This is a very common feature request from the community. 

We should design this API thoughtfully and make it into a real scenario.  (For example people who are migrating from Lerna often ask for an equivalent of the [--since](https://github.com/lerna/lerna/tree/main/core/filter-options#--since-ref) parameter.  But without seeing a fully worked out use case, it's hard to say whether that is a good design or not.)

How complicated is your script that uses this API?  Maybe your PR could add a simplified demo project under `build-tests` so we can discuss its approach alongside this API, and then the website docs can be based on it.

Reply:
I've added a simple build test. The only API it actually uses is getChangedProjectsAsync.